src/components/sync-connected-sites.tsx

```diff
@@ -248,37 +250,44 @@ const SyncConnectedSitesSection = ( {
 									! isPushError &&
 									! pushState.isInProgress &&
 									! pushState.hasFinished && (
-										<div className="flex gap-2 pl-4 ml-auto shrink-0 h-5">
-											<Button
-												variant="link"
-												className="!text-black hover:!text-a8c-blueberry"
-												onClick={ () => {
-													const detail = connectedSite.isStaging
-														? __(
-																"Pulling will replace your Studio site's files and database with a copy from your staging site."
-														  )
-														: __(
-																"Pulling will replace your Studio site's files and database with a copy from your production site."
-														  );
-													showPullConfirmation( () => pullSite( connectedSite, selectedSite ), {
-														detail,
-													} );
-												} }
-												disabled={ isAnySitePulling || isAnySitePushing }
-											>
-												<Icon icon={ cloudDownload } />
-												{ __( 'Pull' ) }
-											</Button>
-											<Button
-												variant="link"
-												className="!text-black hover:!text-a8c-blueberry"
-												onClick={ () => handlePushSite( connectedSite ) }
-												disabled={ isAnySitePulling || isAnySitePushing }
-											>
-												<Icon icon={ cloudUpload } />
-												{ __( 'Push' ) }
-											</Button>
-										</div>
+										<Tooltip
+											disabled={ ! isOffline }
+											icon={ offlineIcon }
+											text={ __( 'Pulling or pushing a site requires an internet connection.' ) }
```

Comment:
I have noticed that sometimes we add period (`.`) at the end of tooltip strings, sometimes we omit it.

For instance, in the recent PR (https://github.com/Automattic/studio/pull/684) with the last push / pull time, we don't add the dot.

Reply:
Yeah I noticed that one as well. Let's remove the `.` to ensure that we are consistent with other tooltips such as `Open settings` etc.

Reply:
Actually, let's keep the period and add it to the other PR. I checked UI around the app and it seems that we are using it in the sentence formulation e.g. `You're currently online.` vs. not using it for things like `Open settings`. I think it makes sense to keep this pattern.

Reply:
That sounds good to me. 👍🏼